src/coreclr/hosts/inc/coreclrhost.h

```diff
@@ -46,5 +46,22 @@ CORECLR_HOSTING_API(coreclr_execute_assembly,
             unsigned int* exitCode);
 
 #undef CORECLR_HOSTING_API
-                      
+
+struct SequencePointInfo
+{
+    int lineNumber, ilOffset;
+    char16_t* fileName;
```

Comment:
`fileName` is marshaled to managed code as string in [SymbolReader](https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.Debug.SymbolReader/src/System/Diagnostics/Debug/SymbolReader/SymbolReader.cs#L23). `CharSet` can be either `Auto, Ansi, or Unicode`. Is it possible to setup `CharSet` to UTF-8? Perhaps I misunderstood your comment?